script/Ciel/Day 6/06_01_CIEL06_3.txt

```diff
@@ -283,22 +305,35 @@ Without taking the front door, Arach-shi directly enters the living room from th
 [sha:af77e8f7bea5327d78dd2daa845f6eaf0ae52dc6]{
 -- Page 141, Offset 13522. Glued. Mods: @x, @k, @e.
 -- でも、そんなトコロが一流のサバイバーなのかしら？　情弱死すべしな世の中ですし？　目の前のご馳走をホイホイ孕ませてたら<命|タマ>がいくつあっても足りないってコトかしらー！
-But maybe that's the mark of a first-class survivor? After all, we live in a world where it's natural for those who don't know better to die, right? No matter how many <tries|b a l l s> I get, that wouldn't be enough to impregnate the snack in front of me, huh?
+// 情弱 as derog internet illiterate? Can be used for other media sources,
+// keeping it vague here since I assume it's more general in this usage.
+// Also, fucking what is going on
+ But maybe that's the mark of a first-class survivor?
+ We live in a world where you're as good as dead if you can't stay abreast of what's happening, right?
+ It doesn't matter how many lives you have,
+ if you recklessly impregnate the feast in front of you you'll never make it, will you!
 }
 [sha:d841a4ab5aacd91646592e085fa6ae4d3d355a7f]{
 -- Page 141, Offset 13523. Glued. Mods: @x.
 -- キャー、さっすが雑食動物、本能のようにヒニンしてるぅー！」
-Kyaaaa, as expected of an omnivore's instinct to avoid pregnancy!"
+// Assuming that ヒニン is intentionally kana to play on 否認
+// (deny, reject her advances, play hard to get) and 避妊 (contraception)
+ %{i}Kyaaaa%{/i}, just what I'd expect from an omnivore,
+ you've practically got an instinctual sense for contraception!"
 }
 [sha:26363e83a167502b52058294b3246cda4b0185e3]{
 -- Page 142, Offset 13524.
 -- などと言いながら、阿良句氏は俺の座るソファーまで歩み寄ってきた。
-And with that, Arach-sensei walks up to the sofa I am sitting on.
+As she continues to spout off like this,
```

Comment:
"spout off _like this_" sounds a little awkward. simplify to "go off like this" imo